src/scenes/Event/scenes/Oracle/Sidebar.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable */
+import React from 'react';
```

Comment:
old style:
![screen shot 2018-07-20 at 17 19 44](https://user-images.githubusercontent.com/7886068/43030077-25d9deec-8c41-11e8-8cae-b42366f283c8.png)

the one in PR:
![screen shot 2018-07-20 at 17 20 10](https://user-images.githubusercontent.com/7886068/43030080-31b251a4-8c41-11e8-9881-f482ebd694d1.png)


when moving to styled-components, don't sacrafice the styles, it's hard to notice and come back to fix >.<